src/InterventionBackend.php

```diff
@@ -858,4 +880,24 @@ protected function isStreamReadable($stream)
         $meta = stream_get_meta_data($stream);
         return isset($meta['mode']) && (strstr($meta['mode'], 'r') || strstr($meta['mode'], '+'));
     }
+
+    /**
+     * If a file operation is erroring we can quickly eat up disk space with these exceptions.
+     * Therefore we only want to log these errors in production if the logging is enabled.
+     *
+     * We always log when in dev mode to try and alert developers to errors that can exist
+     *
+     * @param Exception $exception
+     */
+    private function logException(Exception $exception)
+    {
+        $isEnabled = Environment::getEnv('InterventionBackend_EnableExceptionLogging')
```

Comment:
I don’t think we need to introduce a new environment variable for this, it could be handled in config (forgive the pseudo-code):

```
if Injector::hasService(Logger.interventionLogger)
    interventionLogger->error()
else
    nullLogger->error()
```

Then the default YAML config could be:

```yml
---
Only:
  environment: dev
---
Injector:
  Logger.interventionLogger:
    logConfigEtc...
```

So on test/live it just hits the null logger and on dev it hits an actual error logger, then projects can opt into logging on test/live with YAML. What do you think?